person.rb

```diff
@@ -1,7 +1,7 @@
-require './nameable'
+require_relative 'nameable'
 
 class Person < Nameable
-  attr_reader :id
+  attr_reader :id, :rentals
```

Comment:
- [ ] Getters and Setters for `@rentals`, the rentals should be `attr_accessor`, kindly fix that 🙏🏾

Reply:
corrected